src/main/java/gregtech/api/util/GTUtility.java

```diff
@@ -790,7 +796,7 @@ public static boolean canSeeSunClearly(@NotNull World world, @NotNull BlockPos b
         return world.isDaytime();
     }
 
-    public static MapColor getMapColor(int rgb) {
+    public static @NotNull MapColor getMapColor(int rgb) {
```

Comment:
Can this go in ColorUtil now? As well as the other color stuff used for prospector stuff with the various map mods.